wordpress-mobile/WordPress-iOS
# Prime for Push: inline primer on notification tab

Refs #9185

This adds the push notification "primer" to the notifications tab (see mockup in #9185). This is intended to give users the opportunity to grant permission to send push notifications at a better time than currently (immediately after logging in).

Currently hidden behind a feature flag.

<img width="563" alt="image" src="https://user-images.githubusercontent.com/517257/39894666-da8abf9c-5464-11e8-8f7e-4e5b39544b94.png">


To test:
> **Note:** this can be hard to test, because of the checks in place to make sure we don't nag. Best bet is to add `true ||` to the two if statements in `setupInlinePrompt()` around line `409`
- ensure that the primer appears when appropriate
- ensure that allowing notifications or dismissing the primer ("Not now") both work
- ensure that the ratings prompt still works (try a `true` on line `416`)

cc: @SylvesterWilmott care to double check the design? (I know you're afk atm… if I miss changes I'll add them to #9185)
